tools/project_template/README.md

```diff
@@ -12,7 +12,7 @@ If you'd like to work on a PowerToy template, make required modifications to `\t
 
 - Add the new PowerToy project to the `src\modules\` folder for all the relative paths to work.
 - For the module interface implementation take a look at [the interface](/src/modules/interface) and
-    [the example PowerToy implementation](/src/modules/example_powertoy/dllmain.cpp)
+    [the example PowerToy implementation](/tools/project_template/ModuleTemplate/dllmain.cpp)

```

Comment:
Please remove ` and
    [the example PowerToy implementation](/src/modules/example_powertoy/dllmain.cpp)	    [the example PowerToy implementation](/tools/project_template/ModuleTemplate/dllmain.cpp)`